<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="admin_title_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>832</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>관리자 페이지</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="locker_list_title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>사물함 목록 확인</string>
   </property>
  </widget>
  <widget class="QLabel" name="assign_title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>사물함 할당</string>
   </property>
  </widget>
  <widget class="QLabel" name="assign_locker">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>선택한 사물함 : </string>
   </property>
  </widget>
  <widget class="QComboBox" name="student_combo_box">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>530</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="maxVisibleItems">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="assign_student">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>할당할 학생 : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="setStudentToLockerBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>할당하기</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="return_title">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>사물함 반납</string>
   </property>
  </widget>
  <widget class="QLabel" name="return_locker">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>480</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>선택한 사물함 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="return_student">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>520</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>사용중인 학생 : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>580</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>반납하기</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="student_list_title">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>90</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>학생 목록 확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getStudentDBBtn">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>90</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>확인하기</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="student_list">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>170</y>
     <width>256</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="student_name">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>250</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>이름 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="check_usage">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>290</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>사용 여부 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="usage_locker">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>330</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>사용중인 사물함 : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="lockerspace1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>A-1</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="lockerspace2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>A-2</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="lockerspace3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>A-3</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="lockerspace4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>A-4</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="lockerspace5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>A-5</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="lockerspace6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>B-1</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="lockerspace7">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>B-2</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="lockerspace8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>B-3</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="lockerspace9">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>B-4</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="lockerspace10">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>B-5</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
